src/common/ui/code.vue

```diff
@@ -486,6 +494,27 @@ $selectionDarkBg: rgba(73, 72, 62, .99);
   }
 }
 
+/* CodeMirror show-hints fix to work here */
+.CodeMirror-hints {
+  z-index: 9999;
+  /* consistent font with the rest, copied from .monospace-font rule from style/style.css */
+  font-family: "Fira Code", Menlo, Monaco, Consolas, "Liberation Mono"
+                , "Courier New", Courier, monospace;
+}
+
+@media (prefers-color-scheme: dark) {
+  .CodeMirror-hints {
+    background: var(--bg);
+  }
+  .CodeMirror-hint {
+    color: var(--fg);
+  }
+  li.CodeMirror-hint-active {
+    background:white ;
+    color: #08f;
```

Comment:
Dark themes don't use the white background color usually so this should be

```css
  li.CodeMirror-hint-active {
    background: var(--fill-6);
  }
```

Reply:
just to make sure: the background is for the selected autocomplete option. In this case, it corresponds the white background of selected `querySelector` in dark theme screen shot. 

Using `--fill-6`, a light grey against dark background make the highlighting not too clear:

![image](https://user-images.githubusercontent.com/250644/87251097-10509180-c41e-11ea-9dba-944a823f59ec.png)

Reply:
The foreground color should be specified as `var(--fg)`

Reply:
I changed selected foreground to `var(--fg)`. I feel using `var(--fg)` does not standout enough though.

####  Use `--fg` as foreground
```css
  li.CodeMirror-hint-active {
    background: white ;
    color: --var(fg);
  }
```
![image](https://user-images.githubusercontent.com/250644/87256611-0ee88e80-c449-11ea-9d20-d4fce6072eb8.png)


####  initially in the pull request
```css
  li.CodeMirror-hint-active {
    background: white ;
    color: #08f;
  }
```

![image](https://user-images.githubusercontent.com/250644/87256627-35a6c500-c449-11ea-9803-b2d359f411b2.png)

Reply:
Now it's washed out and not readable at all.
Let's go with a simple inversion:

```css
background: var(--fg);
color: var(--bg);
```

Reply:
Just committed the changes:

![image](https://user-images.githubusercontent.com/250644/87337597-8f19fd00-c4f8-11ea-8caf-1ee5b0bfa338.png)